src/yanfeishao/cs555/utils/DateUtils.java

```diff
@@ -117,6 +132,16 @@ private void parseChildDateError(Set<String> result, FamilyEntity familyEntity,
                             }
                         }
                     }
+                    break;
+                    case ErrorCode.US12: {
+                        if (CommonUtils.isNotNull(childBirthDate)) {
+                            if (CommonUtils.isNotNull(wifeBirthDate) && (compareDate(childBirthDate, wifeBirthDate, DateType.YEAR) >= 60)) {
+                                result.add(String.format(FormatterRegex.ERROR_PERSON + ErrorInfo.US12_PERSON, prefix, KeywordsConstant.MOTHER, familyEntity.getMother().getIdentifier(), familyEntity.getMother().getName(), wifeBirthDate, KeywordsConstant.MOTHERAGE, KeywordsConstant.HER, child.getIdentifier(), child.getName(), childBirthDate, familyEntity.getIdentifier()));
+                            } else if (CommonUtils.isNotNull(husbandBirthDate) && (compareDate(childBirthDate, husbandBirthDate, DateType.YEAR) >= 80)) {
```

Comment:
You declare the 60 and 80 as constant variable, you need using this here
e.g. Integer.parseInt(KeywordsConstant.MOTHERAGE)